mbed_lstools/lstools_base.py

```diff
@@ -28,7 +28,7 @@
 class MbedLsToolsBase:
     """ Base class for mbed-lstools, defines mbed-ls tools interface for mbed-enabled devices detection for various hosts
     """
-    def __init__(self):
+    def __init__(self, **kwargs):
```

Comment:
If this extends in future we don't have to keep adding arguments in this class and the derived tree. It is fare to have a transparent mechanism between ```main()``` and this main class for the options in my opinion.